app/Providers/TelescopeServiceProvider.php

```diff
@@ -0,0 +1,58 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Brewmap\Providers;
+
+use Brewmap\Eloquent\User;
+use Illuminate\Support\Facades\Gate;
+use Laravel\Telescope\IncomingEntry;
+use Laravel\Telescope\Telescope;
+use Laravel\Telescope\TelescopeApplicationServiceProvider;
+
+class TelescopeServiceProvider extends TelescopeApplicationServiceProvider
```

Comment:
How this class is added to the system? I cannot see it in any other provider or `config/app.php`. It should be dependent on `config("telescope.enabled")` value, right?

Reply:
It would be nice if you could provide simple scenarios to test if telescope (routes, provider, whatever) is enabled in local environment and disabled in production.

Reply:
Done, but I didn't find any better solution of faking Laravel config state than just simply bootstrapping application once again in that manner:

```
/**
* @Given telescope is enabled
*/
public function telescopeIsEnabled(): void
{
    $app = require "/application/bootstrap/app.php";
    $app->afterBootstrapping(LoadConfiguration::class, function (Application $app): void {
        $app->make('config')->set('telescope.enabled', true);
    });
    $app->make(Laravel::CONSOLE_KERNEL_INTERFACE)->bootstrap();
}
```

However, I added `LaravelBooter` class to make booting Laravel logic consistent and reusable in both `BehatExtension` `load()` method and in contexts, so it can be specified in scenarios this way:
```
Given application is booted with config:
| config            | value |
| app.env           | local |
| telescope.enabled | true  |
```